packages/core/src/modules/application/queries/list-applications.ts

```diff
@@ -36,6 +36,11 @@ export async function listApplications({
       return qb.where('applications.status', '=', status);
     });
 
+  const orderDirection =
+    status === 'accepted' || status === 'rejected' || status === 'all'
+      ? 'desc'
+      : 'asc';
```

Comment:
I love this solution! 🔥

Nit: Since the `pending` applications are the only time we'll need to sort in descending order, can we flip this logic?

```suggestion
  const orderDirection = status === 'pending' ? 'asc' : 'desc';
```

Reply:
Oh nice! That is definitely sweeter 😌